docs/portal/developer-portal/admin_publication.json

```diff
@@ -6,12 +6,12 @@
 "content_class": "html-default",
 "source_system": "git",
 "source_system_id": "https://github.com/Cray-HPE/docs-uan",
-"source_system_version": "v2.7.0",
+"source_system_version": "v@product_version@",
 "lifecycle": "DRAFT",
 "products": ["1013083813"],
-"product_version": "UAN 2.7.0",
+"product_version": "UAN @product_version@",
 "col_id": "crs8033_@docid_suffix@",
-"full_title": "HPE Cray Supercomputing User Access Node (UAN) Software Administration Guide (S-8033)",
+"full_title": "HPE Cray Supercomputing User Access Node (UAN) Software Administration Guide (S-8033) @product_version@",
```

Comment:
shouldn't this be `@product_version_short@`? Does `@product_version@` give you the longer string with the commit and `-dirty` appended? This longer version is fine for the metadata that won't be (directly) human viewable. This metadata field is what's visible as the publication title on HPESC. We need to make the same change to `install_publication.json`

Reply:
`-dirty` means there are uncommitted changes. So you saw that on your local build because you hadn't committed the changes to your branch. Here's the description for `version.sh`

```
# Tags should be of the format v[semver-compatible version]

# If the tag is v1.2.3:
#   and the pipeline runs this, then the version will be 1.2.3
#   and this is run locally, then the version will be 1.2.3
#   and this is run locally with committed changes, then the version will be 1.2.3-{commits}-{git hash}
#   and this is run locally with uncommitted changes, then the version will be appended with '-dirty'
```

When this is built properly with a tag, it should either be the clean "2.7.1" or the short version "2.7".

Reply:
you're right. I redid the process after checking out the v2.6.2 tag and `@product_version@` was substituted correctly. `@product_version@` is what I want in the publication metadata and titles, not `@product_version_short@`